scalableminds/webknossos
# Show the mapping view iff a segmentation layer exists

In discussion with sloomba, I found out that some users don't even know the "mappings view" even though they could make use of it. At the moment, the mappings view is only shown for volume tracings, which doesn't make much sense, since mappings (and the displayed cell ids) are of relevance as soon as wk renders a segmentation layer.
I also also took the liberty to rename the view to Segmentation as I find this more fitting.

### Mailable description of changes:
 - When viewing/tracing a dataset which has a segmentation layer, a "Segmentation" tab will shown in WebKnossos. This "Segmentation" tab is equal to the old "Mappings" tab which was only shown for volume tracings. The Tab shows information about certain cell ids as well as activated mappings.

### Steps to test:
- open different tracings (with and without segmentation layer) in different modes (view, skeleton, volume) and verify that the segmentation view is always shown when the segmentation layer exists

### Issues:
- fixes #2634 

------
